src/coreclr/vm/qcall.h

```diff
@@ -20,12 +20,6 @@
 // GC mode like a normal P/Invoke. These two features should make QCalls easier to write reliably compared to FCalls.
 // QCalls are not prone to GC holes and GC starvation bugs that are common with FCalls.
 //
-// QCalls perform better compared to FCalls w/ HelperMethodFrame. The QCall overhead is about 1.4x less compared to
```

Comment:
Can this be some clarification about QCall can trigger GC and EH since it won't be possible in FCall?

Reply:
Technically there are other options than QCall, eg. `SoftwareExceptionFrame` or `DynamicHelperFrame`. The first one is quite specific, the other one is somewhat generic and could be used as a replacement if done right but it requires assembly stubs. That's what the new allocation helpers do in the slow path.